docs/development/plugin-triggers.md

```diff
@@ -164,7 +164,7 @@ esac
 ### `docker-args-build`
 
 - Description:
-- Invoked by: `dokku build`
+- Invoked by: `dokku_build`
```

Comment:
We should probably say these are functions within dokku if anything. I would otherwise leave them as is.